src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("Figure: circle, ")
+                        .append("area: ").append(getArea()).append(" sq.units, ")
+                        .append("radius: ").append(radius).append(" units, ")
+                        .append("color: ").append(color);
```

Comment:
Accessing the `color` field directly is fine since it's protected and `Circle` is a subclass of `Figure`. However, if encapsulation is a concern, consider using a getter method for accessing `color`.